README.md

```diff
@@ -83,4 +83,7 @@ The following table contains the required ENV variables that the function requir
 |---------------------------------|-----------------------------------------------------|--------|
```

Comment:
I've removed those variables since they are leftovers from the previous task, where we were configuring all the variables to generate the token for the magic link.